pandevice/base.py

```diff
@@ -1873,7 +1876,10 @@ def parse_xml(self, xml):
 
     def element_str(self):
         """The XML form of this object (and its children) as a string."""
-        return ET.tostring(self.element())
+        val = ET.tostring(self.element())
+        if hasattr(val, "hex"):
```

Comment:
Possible other way to handle this instead of checking for "hex" attribute:

    try:
        # Python3
        return ET.tostring(self.element(), encoding='unicode')
    except LookupError:
        # Python 2
        return ET.tostring(self.element())

The returned string is of type 'str' for both python2 and python3.  Not sure if this is better or worse, up to you.